edx_rest_api_client/auth.py

```diff
@@ -17,18 +17,19 @@ def __init__(self, username, full_name, email, signing_key, issuer=None, expires
         self.tracking_context = tracking_context
 
     def __call__(self, r):
-
+        now = datetime.datetime.utcnow()
         data = {
             'username': self.username,
             'full_name': self.full_name,
             'email': self.email,
+            'iat': now,
```

Comment:
I know this is old so might not be needed.  But should the 'exp' be set to 'now' here and then be overridden below if there is a delay before expiring?  ie, if 'exp' doesn't exist, is it assumed to expire immediately or never?